storage/innobase/include/log0recv.h

```diff
@@ -57,7 +57,7 @@ struct recv_addr_t;
 #ifdef XTRABACKUP
 /** map of tablespace_id, that would be full scan during backup with
  * pagetracking*/
```

Comment:
Can you please fix the comment.

List of tablespaces that should be copied fully. Even if paget racking is active, the tablespaces in this list will be fully copied to backup directory. This set/map is populated by the redo scan phase (by a single thread) and after the parsing is over, we only use it for reads.  Hence, it is not required to protect with mutex.

Reply:
Done